### PR TITLE
koordlet:fix import typo

### PR DESCRIPTION
**Pull request by halegreen**
### Ⅰ. Describe what this PR does
fix koordlet.go import typo

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

**Comment by halegreen** [comment]
PTAL @zwzhang0107 @jasonliu747

File: pkg/koordlet/koordlet.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 	"os"
 	"time"
 
-	topologyclientset "github.com/k8stopologyawareschedwg/noderesourcetopology-api/pkg/generated/clientset/versioned"
```
Comment (jasonliu747) [inline pkg/koordlet/koordlet.go:25]: actually, I don't see any typos here.

Comment (halegreen) [reply pkg/koordlet/koordlet.go:25]: yes, actually it's not  typos,   thinking "github.com" imports should be gathered together ?

Comment (jasonliu747) [reply pkg/koordlet/koordlet.go:25]: we usually follow the convention below:

```
import (
  // stdlib
  "fmt"
  "os"

  // third party packages
  "go.uber.org/atomic"
  "golang.org/x/sync/errgroup"

  // project packages
  "github.com/user/project"
)
```

**Comment by zwzhang0107** [comment]
it seems not necessary, since the ci-lint fails

**Pull request closed without merging**